doc/getting-started/scopes.rst

```diff
@@ -0,0 +1,32 @@
+Scopes
```

Comment:
Is "Scopes" the right title for this page? I'm not sure what the word refers to. Maybe if you tried to summarize what the page is about you would find a more descriptive title?